mne/decoding/transformer.py

```diff
@@ -88,7 +88,7 @@ def fit(self, epochs_data, y):
         return self
 
     def transform(self, epochs_data, y=None):
-        """Standardize data across channels.
+        """Standardize data across channels. Operates in-place.
```

Comment:
Yes. I wasn't sure where this ``Scaler`` is used. Should I make it copy the data?
Also the ``y`` here is not used, but there was some comment elsewhere about compatibility with scikit learn, so I guess this can be used as a callback or something.